keda/crds/scaledjobs.keda.sh.yaml

```diff
@@ -1,8 +1,9 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I can't find the issue exactly but think we've discussed not to bump the version here as it enforced Kubernetes 1.16, right @zroubalik?

Reply:
Tracked on https://github.com/kedacore/charts/issues/33 here, but there was another issue as well